salt/utils/mac_utils.py

```diff
@@ -321,7 +326,7 @@ def _available_services():
                     # the system provided plutil program to do the conversion
                     cmd = '/usr/bin/plutil -convert xml1 -o - -- "{0}"'.format(
                         true_path)
-                    plist_xml = salt.modules.cmdmod.run(cmd, output_loglevel='quiet')
+                    plist_xml = __salt__['cmd.run'](cmd, output_loglevel='quiet')
```

Comment:
This should also be using @gtmanfred's suggestion. See salt/grains/core.py for examples. That file maps `_run_all_quiet` into `__salt__ for ease of use, you may wish to do the same here or you can just call it directly.

This will also keep you from needing to invoke the loader to get all of the functions, which will speed up the utils loader.

Reply:
thanks @gtmanfred  and @terminalmage ! this is sooo much easier. working on the fix now :)